### PR TITLE
fix: /-/all/since had been redirect to /-/all/static/today.json

### PR DESCRIPTION
**Pull request by fengmk2**
data format from map change to array

http://weibo.com/1400854834/Cz8OkaAQL

### CODE REVIEW

**Comment by fengmk2** [comment]
Recover now!

image

image

File: services/npm.js [hunk 1]
```diff
@@ -70,12 +70,22 @@ exports.get = function* (name) {
 };
 
 exports.getAllSince = function* (startkey, timeout) {
+  return yield exports.getAllToday(timeout);
```
Comment (dead-horse) [inline services/npm.js]: 两个 return？ 下面的不用了？

Comment (fengmk2) [inline services/npm.js]: 后面又改了一下

**Pull request merged**